src/Helper/ConfigurationHelper.php

```diff
@@ -214,7 +219,33 @@ protected function loadFromSpecificFile($filePath)
         }
 
         $this->detectedPath = $filePath;
-        return include $filePath;
+
+        try {
+            $configuration = $this->getConfigurationArray($filePath);
+        } catch (\UnexpectedValueException $e) {
+            throw $e;
+        }
+
+        return $configuration;
+    }
+
+    /**
+     * @param string $filePath
+     * @return mixed
+     * @throws \UnexpectedValueException
+     */
+    protected function getConfigurationArray($filePath)
+    {
+        $extension = strtolower(pathinfo($filePath, PATHINFO_EXTENSION));
+        if ( $extension === 'php') {
+            $configuration = include $filePath;
+        } elseif ($extension === 'yml') {
```

Comment:
I think that `.yaml` is also a valid extension, so it would be good to support it here!